mautic/mautic
# Fix issue #10675 - respect 'Reply to Contact'

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [X]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL 
| Issue(s) addressed                     | Fixes #10675 



#### Description:
Fixes bug that was introduced in the Send form results action and with Reply to contact set to true.

Added check to see if reply-to is set and only set if empty.


#### Steps to test this PR:


1. Set system-wide reply-to value.
1. Create From with Send form results action.
1. Set Reply to contact set to false.
1. Submit form - verify that reply-to is system-wide setting.
1. Edit from Step 2 and set Reply to contact set to true.
1. Submit form - verify that reply-to is Lead email.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10773"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
